tools/src/tensil/tools/Main.scala

```diff
@@ -41,14 +41,14 @@ object Main extends App {
       .text("Tensil architecture descrition (.tarch) file")
 
     opt[Seq[String]]('o', "output")
-      .valueName("<names>")
+      .valueName("<name>,...")
       .action((x, c) => c.copy(outputNodes = x))
       .text("Optional list of output nodes, defaults to \"Identity\"")
 
-    opt[Int]('b', "batch")
-      .valueName("<integer>")
-      .action((x, c) => c.copy(inputBatchSize = x))
-      .text("Optional size of input batch, defaults to 1")
+    opt[String]('i', "input-shapes")
+      .valueName("<name>[<dim>,...],...")
+      .action((x, c) => c.copy(inputShapes = x))
+      .text("Optional input shapes, defaults to \"[1]\" (batch size of 1)")
```

Comment:
if I specify only one of the input shapes, do the others default to [1]? maybe we should make that clear somehow

Reply:
Added explanation--ptal.

Reply:
Looks good, thanks!